src/refmvs.rs

```diff
@@ -120,11 +119,10 @@ pub struct RefMvsFrame {
     pub mfmv_ref2cur: [i32; 3],
     pub mfmv_ref2ref: [[i32; 7]; 3],
     pub n_mfmvs: i32,
+    pub n_blocks: i32,
```

Comment:
I think this can be a `u32`, since I had made `r_stride` and `n_tile_rows` `u32`s already.

Reply:
```suggestion
    pub n_blocks: u32,
```